### PR TITLE
Auto generate batch of core resources

### PR DESCRIPTION
**Pull request by agualis**
We are going to update the old manually created js resource files in small batches. This is the fifth batch.

### CODE REVIEW

File: src/resources/payment-instruments-resource.js [hunk 1]
```diff
@@ -1,30 +1,25 @@
...
-        },
-
-        update({id, data}) {
-            return apiHandler.patch(`payment-instruments/${id}`, data);
-        },
-
-        deactivate({id}) {
-            return apiHandler.post(`payment-instruments/${id}/deactivation`);
-        },
+      getAll({limit = null, offset = null, expand = null} = {}) {
```
Comment (cesarlevel) [inline src/resources/payment-instruments-resource.js:8]: I'm surprised that there are no filters for this resource

File: src/resources/plans-resource.js [hunk 2]
```diff
@@ -1,31 +1,25 @@
...
-        },
-
-        update({id, data}) {
-            return apiHandler.put(`plans/${id}`, data);
-        },
-
-        delete({id}) {
-            return apiHandler.delete(`plans/${id}`);
-        }
+      getAll({limit = null, offset = null, expand = null} = {}) {
```
Comment (cesarlevel) [inline src/resources/plans-resource.js:8]: We do use filters on plans `/plans?filter=productId` or am I missing something?

File: src/resources/products-resource.js [hunk 3]
```diff
@@ -1,31 +1,25 @@
...
-        },
-
-        update({id, data}) {
-            return apiHandler.put(`products/${id}`, data);
-        },
-
-        delete({id}) {
-            return apiHandler.delete(`products/${id}`);
-        }
+      getAll({limit = null, offset = null, expand = null} = {}) {
```
Comment (cesarlevel) [inline src/resources/products-resource.js:8]: Also filters for products

**Comment by agualis** [comment]
@cesarlevel You are super right. We had forgotten to add those parameters in the `api-definitions`. After merging this [PR](https://github.com/Rebilly/api-definitions/pull/401) the auto-generated resources should include those missing params.

**Comment by agualis** [comment]
> @cesarlevel You are super right. We had forgotten to add those parameters in the `api-definitions`. After merging this [PR](https://github.com/Rebilly/api-definitions/pull/401) the auto-generated resources should include those missing params.

I think they are all fixed now.

**Review by cesarlevel — APPROVED** [review]

**Pull request merged**